### PR TITLE
Fix typo in  RELEASE_NOTES.md

### PR DESCRIPTION
**Pull request by MridulSharma03**
# Summary

- This PR is a (refactoring, bugfix, feature, something else)
- It does the following (modify list as needed):
  - Modifies/refactors (class or method) (how?)
  - Fixes (issue number(s))
  - Adds (specific feature) at the request of (project or person)

# Design review (for API changes or additions---delete if unneeded)

On (date), we reviewed this PR.  We discussed the design ideas:

1. First idea or goal
2. Second idea
3. Third idea

This PR implements 1. and 3.  It leaves out 2. for the following reasons

- (impractical)
- (too big)
- (not a good idea anyway)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -622,15 +622,15 @@ Notable changes include:
       * New OpenMP execution policies added to support static, dynamic, and 
         guided scheduling.
       * Added support for const iterators to be used with RAJA scans.
-      * Support for bitwise and and or reductions have been added.
+      * Support for bitwise and or reductions have been added.
```
Comment (adayton1) [inline RELEASE_NOTES.md:625]: Support for bitwise "and" and "or" reductions has been added

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -199,7 +199,7 @@ Notable changes include:
   * A compilation bug was fixed in the new reduction interface support for 
     OpenMP target offload.  
   * An issue was fixed in AVX compiler checking logic for RAJA vectorization
-    intrinsics capabilities.
+    intrinsic capabilities.
```
Comment (adayton1) [inline RELEASE_NOTES.md:202]: I think we actually want the noun rather than the adjective

File: RELEASE_NOTES.md (hunk 1 above)
Comment (adayton1) [reply RELEASE_NOTES.md:625]: Though I'm not sure how markdown treats quotes.

Comment (rhornung67) [inline RELEASE_NOTES.md:625]:
```suggestion
      * Support for bitwise "and" and "or" reductions have been added.
```

Double quotes in markdown files are rendered properly when you open a markdown file in GitHub and also in the way it appears in a browser.

**Pull request closed without merging**